config/locales/views.fr.yml

```diff
@@ -1170,6 +1171,7 @@ fr:
       export_not_allowed: 'Tu n''as pas le droit d''exporté les abonnés de cet abonnement, car ils contiennent des rôles ou des activités qui ne font pas partie de ton domaine d''autorisation.'
     subscription:
       only_persons_with: 'Seulement personnes avec:'
+      persons_excluded_with: 'Personnes exclues avec:'
```

Comment:
usually we only translate the German version and the translation is done by translators. but of course, if it's short text and you know what you're doing, then do it :)